# Generated by Django 2.2 on 2019-06-23 06:47

import datetime
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('contact', '0003_auto_20190622_2137'),
    ]

    operations = [
        migrations.CreateModel(
            name='AcceptedProject',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('message', models.CharField(default='has accepted your connection request.', max_length=30)),
                ('receiver_id', models.IntegerField()),
                ('receiver_name', models.CharField(max_length=30)),
                ('receiver_picture', models.ImageField(upload_to='')),
                ('project_name', models.CharField(max_length=30)),
                ('user_id', models.IntegerField()),
                ('user_name', models.CharField(max_length=30)),
                ('contact_date', models.DateTimeField(blank=True, default=datetime.datetime.now)),
            ],
        ),
        migrations.CreateModel(
            name='AcceptedUser',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('message', models.CharField(default='has accepted your connection request.', max_length=30)),
                ('receiver_id', models.IntegerField()),
                ('receiver_name', models.CharField(max_length=30)),
                ('receiver_picture', models.ImageField(upload_to='')),
                ('user_id', models.IntegerField()),
                ('user_name', models.CharField(max_length=30)),
                ('contact_date', models.DateTimeField(blank=True, default=datetime.datetime.now)),
            ],
        ),
    ]
